Makefile

```diff
@@ -7,7 +7,7 @@ SHELL := /bin/bash
 LOCAL := $(PWD)/usr
 PATH := $(LOCAL)/bin:$(PATH)
 
-# Path to the .traineddata directory to start finetuning from. Default: $(LOCAL)/share/tessdata
+# Path to the .traineddata directory with BEST traineddata to start finetuning from. Default: $(LOCAL)/share/tessdata
```

Comment:
Is there a better name than `BEST traineddata`? Google used `traineddata_best` for model files which use the float or double C datatype and which are required for training.

What about "with traineddata suitable for training (for example from tesseract-ocr/tessdata_best)" (and lengthy comment split in two lines)?

Reply:
Changed via https://github.com/tesseract-ocr/tesstrain/pull/118/commits/9bdfe3fa7ffabafe4d1a658b0503d965671f9a5d

Reply:
Thanks! Any other changes needed?